### PR TITLE
[FAD-70] Build Dashboard Totals Overview

### PR DESCRIPTION
**Pull request by v1n33th**
### Description

Added dashboard total component and related files

- [x] `CHANGELOG` has been updated (_if appropriate_)
- [x] Environment variables updated (_if appropriate_)

### CODE REVIEW

**Comment by v1n33th** [comment]
Images

With Button
Screen Shot 2022-09-09 at 3 14 56 PM

Without Button
Screen Shot 2022-09-09 at 3 15 27 PM

File: packages/client3/components/dashboard/total/dashboard-total.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        "grid grid-cols-4 gap-4 bg-green-100 px-7 py-12",
+        "bg-img-free-artist bg-center bg-no-repeat"
+      )}
+    >
+      <div>
+        <BodyText size="normal" className="text-white/30">
+          Total Raised
+        </BodyText>
+        <Heading level={2} className={hasData ? "text-white" : "text-dark-70"}>
+          ${totalRaised ?? 0}
```
Comment (luansievers) [inline packages/client3/components/dashboard/total/dashboard-total.tsx:30]: I think you can format those values with `formatCrypto`

File: packages/client3/components/dashboard/total/dashboard-total.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        </Heading>
+      </div>
+      <div>
+        <BodyText size="normal" className="text-white/30">
+          Total Earned
+        </BodyText>
+        <Heading level={2} className={hasData ? "text-white" : "text-dark-70"}>
+          ${totalEarned ?? 0}
```
Comment (luansievers) [inline packages/client3/components/dashboard/total/dashboard-total.tsx:38]: Same here

**Review by luansievers — CHANGES_REQUESTED** [review]

**Review by sachasmart-weavik — APPROVED** [review]
Nice man. 

I was OCD and checked the image, tinyPNG'd it and pushed it.

**Review by luansievers — APPROVED** [review]

**Pull request merged**